ansible/group_vars/all/defaults.yaml

```diff
@@ -5,11 +5,11 @@ python_path: ""
 # If it's not there, the kubernetes_full_version will have "None" for a version number.
 #
 # IMPORTANT When you update kubernetes_version, also update crictl_version.
-kubernetes_version: "1.30.5"
+kubernetes_version: "1.31.3"
 kubernetes_major_minor: "{{ (kubernetes_version.split('.') | map('trim'))[:2] | join('.') }}"
 kubernetes_semver: "v{{ kubernetes_version }}"
 
-kubernetes_cni_version: "1.4.0"
+kubernetes_cni_version: "1.6.0"
```

Comment:
In the bundle the version is 1.5.1, which version is the correct one?

https://github.com/mesosphere/dkp-release/blob/917152f4c3c8f83727d4ffcf482fe77d4f1f73dc/k8s-build/package_rpms.sh#L43

Reply:
@SandhyaRavi2403 this version needs to match the version built in the bundle.
If I follow https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/ with:
```
cat <<EOF | tee /etc/yum.repos.d/kubernetes.repo
[kubernetes]
name=Kubernetes
baseurl=https://pkgs.k8s.io/core:/stable:/v1.31/rpm/
enabled=1
gpgcheck=1
gpgkey=https://pkgs.k8s.io/core:/stable:/v1.31/rpm/repodata/repomd.xml.key
exclude=kubelet kubeadm kubectl cri-tools kubernetes-cni
EOF
```

The version is `1.5.1`.
```
 kubernetes-cni                                                                                   aarch64                                                                          1.5.1-150500.1.1                                                                             kubernetes                                                                          6.3 M
```